package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ts-standard",
   "description": "TypeScript Standard Style based on StandardJS",
-  "version": "12.0.2",
+  "version": "12.0.3",
```

Comment:
Now that we automate it, I think we should not increase the version ourselves.
I think we should use `0.0.0-development` instead.

https://github.com/semantic-release/semantic-release/blob/18bd0c490483008230c57ae6a73d07a09dd00d0d/package.json#L4

Reply:
`semantic-release` will bump the version for you, and `@semantic-release/git` will commit and push the version bump to the git repo (This is why the version bump happened, as I was testing semantic-release). The version won't have to be increased manually. :smile: 

You can see this in action in a test repo I made; here's an example commit of what it will do: 
https://github.com/Maytha8/ts-standard-test/commit/b00d5b3fefff87dac6bf0c4f0dccf3c11d116156#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519
Here you can see that it knew what the next version was and bumped it.

The only reason `semantic-release` have their own version set to `0.0.0-development` is because they don't use `@semantic-release/npm` on their own project (it is merely a default dependency).

If you'd like this behavior of leaving the version at `0.0.0-development`, then we could remove package.json from the `assets` property of `@semantic-release/git` in .releaserc.yml.